src/Data/UnityEditor.hs

```diff
@@ -1,4 +1,6 @@
 {-# LANGUAGE DeriveGeneric #-}
+{-# OPTIONS_GHC -fconstraint-solver-iterations=10 #-}
```

Comment:
Yes, I had a build error using GHC 9.8.2. I think it has something to do with the use of QuickCheck, Generics and huge type lists, as the error message suggests. Setting the iteration count up after searching the web for the error solved it for me. It may be not the best solution though. 
``````
UnityEditor.hs:1:1: error: [GHC-95822]
    solveWanteds: too many iterations (limit = 4)
      Unsolved: WC {wc_simple =
                          [W] $dFiniteSum_a8We {7}:: Test.QuickCheck.Arbitrary.Generic.FiniteSum
                                                   HumanBodyBones
                                                   (GHC.Generics.C1
                                                      (GHC.Generics.MetaCons
                                                         "Hips" GHC.Generics.PrefixI False)
                                                      GHC.Generics.U1
                                                    GHC.Generics.:+: (GHC.Generics.C1
......
......
......

Reply:
I see. As I've got some compile error with previous test code, I think it is related to that. (Even though I don't know...)